org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -73,6 +73,12 @@
 	 * Preference key used to include the comments during the formatting.
 	 */
 	public static final String JAVA_FORMAT_COMMENTS = "java.format.comments.enabled";
+	/**
+	 * Specifies filter applied on projects to exclude some file system objects
+	 * while populating the resources tree.
+	 */
+	public static final String JAVA_RESOURCE_FILTER = "java.import.resourceFilter";
+	public static final String JAVA_RESOURCE_FILTER_DEFAULT = "node_modules|.git";
```

Comment:
should be an actual list

Reply:
The argument is a regular expression (string), not a list.

Reply:
Fixed.